webapp/app.py

```diff
@@ -300,9 +302,7 @@ def utility_processor():
 # Ceph docs
 ceph_docs = Docs(
     parser=DocParser(
-        api=discourse_api,
-        index_topic_id=17250,
-        url_prefix="/ceph/docs",
+        api=discourse_api, index_topic_id=17250, url_prefix="/ceph/docs",
```

Comment:
Do you have a line limit set on prettier?

Reply:
atom seems not to... vs code does.